### PR TITLE
Start page

### PR DESCRIPTION
**Pull request by Mykola15**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -17,6 +18,7 @@ function App() {
         <Route path="/user" exact component={User} />
         <Route path="/login" exact component={Login} />
         <Route path="/registration" exact component={Registration} />    
+        <Route path="/startPage" exact component={Start} />
```
Comment (wogy14) [inline src/App.tsx]: Please set route '/' and rename component and dir with it as 'Main'.

File: src/App.tsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import {Route} from 'react-router-dom';
 import User from './components/User/User';
 import Login from './components/Login_Registration/Login';
 import Registration from './components/Login_Registration/Registration';
+import Start from './components/StartPage/Start';
```
Comment (wogy14) [inline src/App.tsx]: Please set route '/' and rename component and dir with it as 'Main'.

File: src/components/StartPage/Start.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Task from './Task/Task';
+import Navbar from './Navbar/Navbar';
+
+function Start() {
```
Comment (wogy14) [inline src/components/StartPage/Start.tsx]: As I wrote above: change `Start` to `Main`

File: src/components/StartPage/Task/Task.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                    <div className='col-md-4 col-sm-6 task-img'>
+                        <figure><img src={image} alt="task-img"/></figure>
+                    </div>
+                    <div className='col-md-8 col-sm-6 row'>
+                        <div className='col-md-12 task-main-data'>
+                            <div className='task-name'><h1>Some name</h1></div>
+                            <div className='task-participants'><h4>Participants: <i>пше3/10 </i></h4></div>
+                            <div className='task-owner'><h4>Owner: Robert De Niro</h4></div>
+                        </div>
+                        <div className='col-md-12 task-description'> Lorem ipsum dolor sit amet, consectetur adipiscing elit,
```
Comment (wogy14) [inline src/components/StartPage/Task/Task.tsx]: You should to make `task-description` as a child div.

File: src/components/StartPage/Task/Task.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import image from './task.svg';
+import './Task.scss';
+
+function Task() {
+    return (
+        <div className='main-task-block'>
+            <div className='container'>
+                <div className='row'>
+                    <div className='col-md-4 col-sm-6 task-img'>
```
Comment (wogy14) [inline src/components/StartPage/Task/Task.tsx]: You should to make `task-img` as a child div.

File: src/components/StartPage/Task/Task.scss [hunk 6]
```diff
@@ -0,0 +1,16 @@
+.main-task-block {
```
Comment (wogy14) [inline src/components/StartPage/Task/Task.scss:1]: Refactor this file Ctrl + alt + L on WebStorm.

File: src/components/StartPage/Navbar/Navbar.tsx [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Navbar() {
```
Comment (wogy14) [inline src/components/StartPage/Navbar/Navbar.tsx]: Rename to `Sidebar`.

**Review by wogy14 — COMMENTED** [review]
You have 2 unnecessary files `.DS_STORE`.

**Review by wogy14 — APPROVED** [review]

**Comment by KappasTV** [comment]
Resolve conflicts

**Review by KappasTV — APPROVED** [review]

**Pull request merged**